infra/jark-stack/terraform/blueprint.tfvars

```diff
@@ -1,9 +1,23 @@
 name                             = "jark-stack"
 enable_aws_efs_csi_driver        = true
-enable_aws_cloudwatch_metrics    = true
 enable_jupyterhub                = true
 enable_kuberay_operator          = true
 enable_argo_workflows            = true
 enable_argo_events               = true
 enable_argocd                    = true
 enable_ai_ml_observability_stack = true
+
```

Comment:
This will provide full details of what addons are deployed by this stack

Reply:
Yes, it’s a good idea to add this information to all the tfvars files so it’s clear and easy to find. This is the main file most people will look at before deploying the stack. The base infrastructure has a lot of variables, which can be confusing.

Ideally, we should document the optional add-ons on a website or somewhere accessible, so users can easily copy and paste the ones they want into their tfvars file.

Reply:
Yes, we need to ensure this website doc is updated through precommit every time we add a new addon. Similar to terraform docs. 

Somehow link base infra terraform input variables doc to website.

Reply:
I will raise an issue for this so that we can handle this update in the next PR